TravBotSharp/Views/VillageViews/FarmingNonGoldUc.cs

```diff
@@ -0,0 +1,303 @@
+﻿using System;
+using System.Windows.Forms;
+using System.Drawing;
+
+using TravBotSharp.Interfaces;
+using TravBotSharp.Files.Helpers;
+using TravBotSharp.Files.Tasks;
+using TravBotSharp.Files.Tasks.LowLevel;
+
+using TbsCore.Models.VillageModels;
+using TbsCore.Models.SendTroopsModels;
+using TbsCore.Models.MapModels;
+using TbsCore.Models.AccModels;
+
+namespace TravBotSharp.Views
+{
+    public partial class FarmingNonGoldUc : BaseVillageUc, ITbsUc
+    {
+        public FarmingNonGoldUc()
+        {
+            InitializeComponent();
+        }
+
+        private int currentFarmList_index;
+        private Farm currentFarm;
+
+        public void UpdateUc()
+        {
+            var acc = GetSelectedAcc();
+            var vill = GetSelectedVillage(acc);
+
+            if (vill == null) return;
+
+            if (acc.AccInfo.Tribe != null)
```

Comment:
I don't think Tribe can be null, but it can be "Any", which is what you want to check

Reply:
hm I copy this from here https://github.com/Erol444/TravianBotSharp/blob/076039c2b9153095528fa1e75e985a659e5d70a3/TravBotSharp/Views/VillageViews/TroopsUc.cs#L18

Reply:
My bad, I just checked, the Tribe is actually nullable (enum). You are right:)